Makefile

```diff
@@ -107,9 +105,13 @@ CONTROLLER_OCP := $(or ${CONTROLLER_OCP},quay.io/ocpmetal/assisted-installer-con
 
 PLATFORM := $(or ${PLATFORM},baremetal)
 IPV6_SUPPORT := $(or ${IPV6_SUPPORT},true)
-ENABLE_KUBE_API := $(or ${ENABLE_KUBE_API},true)
-
+SERVICE_REPLICAS_COUNT := 3
+ENABLE_KUBE_API := $(or ${ENABLE_KUBE_API},no)
 KUBE_API := $(or ${KUBE_API},no)
+ifeq ($(ENABLE_KUBE_API),true)
+	SERVICE_REPLICAS_COUNT=1
+	KUBE_API=yes
```

Comment:
Not sure about that
This will lead to confusion between ``ENABLE_KUBE_API`` and ``KUBE_API`` which is pretty confusing as of know anyway

Reply:
yep, i thought that if you already set ENABLE_KUBE_API it means that you want to run KUBE_API flow.